jobs/buildhub/lambda_s3_inventory.py

```diff
@@ -21,9 +26,33 @@
 logger = logging.getLogger()  # root logger.
 
 
-async def list_manifest_entries(loop, client, inventory):
+async def initialize_kinto(loop, kinto_client, bucket, collection):
+    """
+    Initialize the remote server with the initialization.yml file.
+    """
+    # Leverage kinto-wizard async client.
+    thread_pool = ThreadPoolExecutor()
+    async_client = AsyncKintoClient(kinto_client, loop, thread_pool)
+
+    initialization_manifest = pkgutil.get_data('buildhub', 'initialization.yml')
```

Comment:
I didn't know about that, maybe we can remove a lot of `zip_safe=False` using that

Reply:
Exactly, that's what I discovered!